runtime/altair/src/lib.rs

```diff
@@ -1591,8 +1591,32 @@ impl_runtime_apis! {
 			let pool = pallet_pool_system::Pool::<Runtime>::get(pool_id)?;
 			pool.tranches.tranche_currency(tranche_loc).map(Into::into)
 		}
+
+		fn portfolio_valuation(pool_id: PoolId) -> Option<Balance> {
+			if !pallet_pool_system::Pool::<Runtime>::get(pool_id).is_some() {
+				return None;
+			}
+
+			match pallet_loans::Pallet::<Runtime>::update_nav_of_pool(pool_id) {
+				Ok((_, value)) => Some(value),
+				Err(_) => None,
+			}
+		}
+
+		fn max_borrow_amount(pool_id: PoolId) -> Option<Balance> {
```

Comment:
I think there's a confusion about this endpoint. This should return the max borrow amount for an individual loan, not for the entire pool (I don't think there's a use case for that in the frontend).